.github/workflows/main.yaml

```diff
@@ -116,40 +116,6 @@ jobs:
       - name: Run e2e tests
         run:  make e2e
 
-  helm_tests:
-    runs-on: ubuntu-22.04
-    timeout-minutes: 30
-    name: helm e2e tests
-    env:
-      GO111MODULE: "on"
-    steps:
-      - name: Checkout
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-        with:
-          fetch-depth: 0
-
-      - name: Install Go
-        uses: actions/setup-go@0aaccfd150d50ccaeb58ebd88d36e91967a5f35b # v5.4.0
-        with:
-          go-version-file: go.mod
-          cache-dependency-path: go.sum
-
-      - name: Setup kubectl
-        uses: marcofranssen/setup-kubectl@d81bf181729cee2bc813ac2ea2f74dec90f01f99 # v1.3.0
-        with:
-          kubectlVersion: ${{ env.KUBECTL_VERSION }}
-
-      - name: Install Helm
-        uses: azure/setup-helm@b9e51907a09c216f16ebe8536097933489208112 # v4.3.0
-        with:
-          version: ${{ env.HELM_VERSION }}
-
-      - name: Install dependences
-        run: make ginkgo kind
-
-      - name: Run helm e2e tests
-        run: make helm-e2e
```

Comment:
didn't depend on other jobs so no need to be in same workflow (allowing for more finegrained control when this part runs see above comment.